sentences/sv/_common.yaml

```diff
@@ -48,13 +48,14 @@ lists:
         out: "fahrenheit"
 expansion_rules:
   name: "{name}[s | n | ns | en ]"
-  area: "{area}[e | me][t | ts | n | ns ]"
```

Comment:
Add the extra variants from main

Reply:
I think that I have all combinations that is in main, and also adding [s] as single usages.